include/zephyr/net/phy.h

```diff
@@ -123,6 +127,26 @@ __subsystem struct ethphy_driver_api {
  * @endcond
  */
 
+/**
+ * @brief      Get the MAC address of the ethernet device given as a
+ *             port number parameter.
+ *
+ * @param[in]  dev     PHY device structure
+ * @param[in]  port    Ethernet port
+ * @param      mac     Buffer to store readed MAC address
```

Comment:
s/readed/the read/

I understand from the API description, that the user provides a pointer to buffer where the mac address is copied. Could this be emphasized in the description, also isn't this too a `[in]` parameter like the others.

Reply:
Yes, this is an `[in]` parameter, but according to the next comment, it should be a pointer to 64-bit buffer rather